x-pack/plugins/reporting/server/routes/lib/jobs_query.ts

```diff
@@ -33,7 +33,7 @@ function getSearchBody(body: SearchRequest['body']): SearchRequest['body'] {
 }
 
 export type ReportContent = Pick<ReportSource, 'status' | 'jobtype' | 'output'> & {
-  payload?: Pick<ReportSource['payload'], 'title'>;
+  payload?: Pick<ReportSource['payload'], 'title' | 'locatorParams'>;
```

Comment:
I might not have the most up-to-date information, but I think we were wanting to limit the number of things we expose to the client since this is mapped to another object before being sent over the wire.

I'll add that this is probably going to change since we are going to remove the additional client side call to fetch the locators for now.

Reply:
There might not be places in the code that use a `ReportContent` object to reference other payload fields that are not listed. I am guessing the fields that are listed out are the ones that are referenced somewhere.